src/main/java/org/sonar/plugins/scmstats/HgScmAdapter.java

```diff
@@ -86,7 +86,7 @@ public ChangeLogHandler getChangeLog(DateRange dateRange) {
       LOG.info("Getting change log information for %s\n", baseDir.getAbsolutePath());
       if (!hgRepo.initFrom(baseDir)) {
         throw new HgRepositoryNotFoundException(
-                String.format("Can't find repository in: %s\n",baseDir.getAbsolutePath()));
+                String.format("Can't find repository in: %s%n",baseDir.getAbsolutePath()));
```

Comment:
%n should be used in place of \n to produce the platform-specific line separator

more info can be found at https://dev.eclipse.org/sonar/rules/show/squid:S2275